src/main/kotlin/lotto/domain/Prize.kt

```diff
@@ -1,20 +1,20 @@
 package lotto.domain
 
-enum class Prize(val matchedCount: Int, val shouldBonusMatch: Boolean, val reward: Long) {
+enum class Prize(val shouldMatchedAtLeast: Int, val shouldBonusMatch: Boolean, val reward: Long) {
     FIRST(6, false, 2_000_000_000L),
     SECOND(5, true, 30_000_000L),
     THIRD(5, false, 1_500_000L),
     FOURTH(4, false, 50_000L),
-    FIFTH(3, false, 5_000L);
+    FIFTH(3, false, 5_000L),
+    NOTHING(0, false, 0L);
```

Comment:
```suggestion
    FIFTH(3, false, 5_000L),
    NOTHING(0, false, 0L),
    ;
```

small suggestion (you might already know)
simple trick to make it flexible to add a new line